docs/fundamentals/code-analysis/style-rules/ide0200.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "IDE0200: Remove unnecessary lambda expression"
+description: "Learn about code analysis rule IDE0200: Remove unnecessary lambda expression"
+ms.date: 07/19/2023
+ms.topic: reference
+f1_keywords:
+- IDE0200
+helpviewer_keywords:
+- IDE0200
+dev_langs:
+- CSharp
+---
+# Remove unnecessary lambda expression (IDE0200)
+
+| Property                 | Value                      |
+| ------------------------ | -------------------------- |
+| **Rule ID**              | IDE0200                    |
+| **Title**                | Remove unnecessary lambda expression |
+| **Category**             | Style                      |
+| **Subcategory**          | Unnecessary code rules     |
+| **Applicable languages** | C# 11+                     |
```

Comment:
Do we want to mention that the rule fires only on C# 11+ to avoid a performance hit: https://github.com/dotnet/roslyn/blob/9999d45981eb74581530c28d297f04ce9ba51004/src/Analyzers/CSharp/Analyzers/RemoveUnnecessaryLambdaExpression/CSharpRemoveUnnecessaryLambdaExpressionDiagnosticAnalyzer.cs#L25-L26

If it feels this might unnecessarily complicate the doc, then it's fine to leave it as-is.

Reply:
I think we should leave it as-is for now.